src/utils/rootkeypackage_utils/inc/aduc/rootkeypackage_types.h

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file rootkeypackage_types.h
+ * @brief rootkeypackage types.
+ *
+ * @copyright Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+#ifndef ROOTKEYPACKAGE_TYPES_H
+#define ROOTKEYPACKAGE_TYPES_H
+
+#include <azure_c_shared_utility/constbuffer.h>
+#include <azure_c_shared_utility/strings.h>
+#include <azure_c_shared_utility/vector.h>
+#include <sys/time.h> // for struct timespec
+
+/**
+ * @brief The root key keytype.
+ */
+typedef enum tagADUC_RootKey_KeyType
+{
+    RSA, /**< RSA keytype. */
```

Comment:
Add EC?

Reply:
Leaving it out for MvP. Crypto agility to add it is an orthogonal effort.